resources/nspawn.go

```diff
@@ -126,6 +142,7 @@ func (obj *NspawnRes) Watch() error {
 	var send = false
 	var exit *error
 
+	defer close(buschan)
```

Comment:
This was an insightful thing to add-- one other thing we need to check first: how do we stop the receiving of signals on that channel *first*-- that's something that I think we forgot to do in v1 of the patch. Without that, we'll cause a panic if we receive a signal after this closes.